ega-data-api-dataedge/src/main/java/eu/elixir/ega/ebi/dataedge/exception/MethodNotAllowedException.java

```diff
@@ -0,0 +1,11 @@
+package eu.elixir.ega.ebi.dataedge.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(HttpStatus.METHOD_NOT_ALLOWED)
+public class MethodNotAllowedException extends RuntimeException {
+    public MethodNotAllowedException(String message) {
+        super(message);
+    }
+}
```

Comment:
@anandmohan777 quick suggestion, this MethodNotAllowed can be useful for more than when services, so maybe as Ashutosh create the common classes for Exceptions do you mind move it there? I think I can be useful for more services than dataEdge. Do you agree? :)